pkg/controller/containerjfr/containerjfr_controller_test.go

```diff
@@ -93,20 +94,20 @@ var _ = Describe("ContainerjfrController", func() {
 				}
 			})
 			It("should create certificates", func() {
-				expectCertificates(client, *controller)
+				expectCertificates(client, controller)
 			})
 			It("should create routes", func() {
-				expectRoutes(client, *controller, false)
+				expectRoutes(client, controller, false)
 			})
 			It("should create persistent volume claim and set owner", func() {
-				expectPVC(client, *controller, false)
+				expectPVC(client, controller, false)
 			})
 			It("should create Grafana secret and set owner", func() {
 				secret := &corev1.Secret{}
 				err := client.Get(context.Background(), types.NamespacedName{Name: "containerjfr-grafana-basic", Namespace: "default"}, secret)
-				Expect(err).To(HaveOccurred())
+				Expect(kerrors.IsNotFound(err)).To(Equal(true))
```

Comment:
The `BeTrue` matcher is probably a better fit for these kind of expectations.
```suggestion
				Expect(kerrors.IsNotFound(err)).To(BeTrue())
```

Reply:
Yes, definitely!